test/e2e/internal/notation/init.go

```diff
@@ -23,6 +23,7 @@ const (
 	envKeyRegistryHost       = "NOTATION_E2E_REGISTRY_HOST"
 	envKeyRegistryUsername   = "NOTATION_E2E_REGISTRY_USERNAME"
 	envKeyRegistryPassword   = "NOTATION_E2E_REGISTRY_PASSWORD"
+	envKeyDomainRegistryHost = "NOTATION_E2E_DOMAIN_REGISTRY_HOST"
```

Comment:
How is this different from `NOTATION_E2E_REGISTRY_HOST` ?

Reply:
NOTATION_E2E_DOMAIN_REGISTRY_HOST is a registry host, separate from localhost, used for testing the --insecure-registry flag.